# OMPL Planner Settings, Mostly for Experience Planning
moveit_ompl:
  use_experience_setup: false
  experience_type: thunder
  planning_time: 60
  use_scratch: true
  use_experience: true
  # but still allow saving experiences
  saving_enabled: true
  sparse_delta_fraction: 0.05 # 0.1 default
